### PR TITLE
💄 Update styles and imports

### PR DESCRIPTION
**Pull request by sweisz**
The pages now have their own css styles.

### CODE REVIEW

**Review by DNaunin — APPROVED** [review]
Good Job! 🚀

File: styles/Skill.module.css [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+    rgba(255, 216, 0, 1),
+    rgba(255, 245, 188, 1) 100%
+  );
+  display: grid;
+  grid-template-columns: auto;
+  height: 100vh;
+  justify-content: center;
+  justify-items: center;
+  align-items: center;
+  padding: 0;
```
Comment (colefriedlaender) [inline styles/Skill.module.css]: Do you really need this? (padding 0)

File: styles/Welcome.module.css [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+  display: grid;
+  grid-template-columns: auto;
+  height: 100vh;
+  justify-content: center;
+  justify-items: center;
+  align-items: center;
+  padding: 0;
+}
+.logo,
+.header {
```
Comment (colefriedlaender) [inline styles/Welcome.module.css:17]: if logo and header have the same styles why dont you give it the same className?

Comment (sweisz) [reply styles/Welcome.module.css:17]: They have the same attribute `center`.

File: styles/Skill.module.css (hunk 1 above)
Comment (sweisz) [reply styles/Skill.module.css]: padding removed

**Pull request merged**